completions/kbsecret.fish

```diff
@@ -0,0 +1,162 @@
+# Things that don't work:
+# -h/--help doesn't display command completions
+# todo subcommands
```

Comment:
Can you mark these lines with `XXX`, `NOTE`, or `TODO`? That should make them more discoverable for future contributors.